data/assets/03_global.css

```diff
@@ -27,29 +27,36 @@ h3 {
 
 /* Page top horizontal bar */
 .main_header {
+  display: inline-block;
+  width: 100%;
   font-size: var(--font-size-large);
   background: var(--blue-0);
   color: var(--white-0);
-  text-align: center;
-  padding-top: 20px;
-  padding-bottom: 40px;
+  padding: 5px 20px;
```

Comment:
I prefer the separate lines for padding configuration, because I can never remeber which is which on the combined line. I've not seen how it looks, but I'm afraid that 5px on top will not be enough.

Reply:
Reverting styling to previous state